crates/hotshot-qc/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "hotshot-qc"
+description = "Quorum certificate instantiations"
+version = { workspace = true }
+authors = { workspace = true }
+edition = { workspace = true }
+rust-version = { workspace = true }
+
+[dependencies]
+ark-bls12-377 = "0.4.0"
+ark-bls12-381 = "0.4.0"
+ark-bn254 = "0.4.0"
+ark-ec = "0.4.0"
+ark-ff = "0.4.0"
+ark-pallas = "0.4.0"
+ark-poly = "0.4.0"
+ark-serialize = "0.4.0"
+ark-std = { version = "0.4.0", default-features = false }
+bincode = { version = "1.1.3" }
+bitvec = { version = "1.0.1", default-features = false, features = ["alloc", "atomic", "serde"] }
+ethereum-types = { version = "0.14.1", features = ["impl-serde"] }
+generic-array = "0.14.7"
+hotshot-types = { path = "../../types" }
+jf-primitives = { git = "https://github.com/EspressoSystems/jellyfish", branch = "hotshot-compat" }
+jf-relation= { git = "https://github.com/EspressoSystems/jellyfish", branch = "hotshot-compat" }
+jf-utils = { git = "https://github.com/espressosystems/jellyfish", branch = "hotshot-compat" }
+serde = { version = "1.0.164", features = ["derive"] }
+typenum = { version = "1.16.0" }
+
+[dev-dependencies]
+hotshot-stake-table = { path = "../hotshot-stake-table" }
+
+[features]
+default = ["parallel"]
+std = ["ark-std/std"]
+parallel = ["jf-primitives/parallel", "jf-utils/parallel"]
+# TODO: (alex) what other features should I add
```

Comment:
`full-ci` is an alias for async-std channels and executor. I'm afraid having `full-ci` here will confuse cargo when it builds `tokio-ci`, since `full-ci` and `tokio-ci` are mutually exclusive. It might be better to just leave these features on by default and have all other crates that use this enable this crate's default features.

Reply:
sounds good, I will just remove `full-ci` here. 

I guess the main question I have, which `just` command to run so that I can run test only on my specific crate while able to compile others?
for example right now I can't just run `just _basic_test` since this is not running my test, I can't just `cargo test -p my-package` because compiler will complain that I gave the wrong feature flag for other crates. 

any idea @DieracDelta or should we add another Just command for what I ask?

Reply:
For now, `just test_async_std_test my_test` will build the workspace and run a test. I'm not sure there's a cargo invocation that will test only that package that works right now. I'm working on changing our build flags per your suggestion to use `RUSTFLAGS`. That should fix this issue. In the mean time maybe we should merge?

Reply:
sounds good, Thanks Justin!